modernism/static/_js/modernism.js

```diff
@@ -126,6 +126,24 @@ const setModalImage = (direction) => {
 
 }
 
+//Get the button
+let mybutton = document.getElementById("myBtn");
+// When scroll down 100px from the top of the document, show the button
+window.onscroll = function() {scrollFunction()};
+
+function scrollFunction() {
```

Comment:
Also here, you could think about the naming. There is no need to name a function "Function", as it is visible by the code anyway. But you could add some meaning to it by giving more details about the function. What does the function do if it is used?

Reply:
I renamed the function.